Configuration/StandardSequences/python/Reconstruction_Data_cff.py

```diff
@@ -16,25 +16,6 @@
         qTest.cleaningThresholds.append(30.)
         qTest.flags.append('HFDigi')
              
-particleFlowRecHitHF.producers[0].qualityTests.append(
```

Comment:
Hello, So the +-5ns is historical since Run I . The question to HCAL is how do they apply the HFDigiTime criterion. If they apply a timing window this should be  equivalent to the +-5 ns window that was implemented in PF for hits with E>30 GeV. I would expect that a +-5ns window could have small effect in 50ns as well. The physics question to the DPG is if they consider safe a +-5 ns window for E>30 GeV in HF hits. If they want it they can keep it in Run I, if they already do it via the severity cuts they can remove it. As I discussed several times in PPD meeting , the selection of hits in PF is something that should be decided by the DPG [as we do in ECAL] so I fully trust them to decide what they want to do